app/models/cart.rb

```diff
@@ -0,0 +1,5 @@
+class Cart < ApplicationRecord
```

Comment:
Could we have Cart be a PORO and then have a table called cart_items that has user_id, item_id, and quantity?

Reply:
PORO created as "basket" so that it keeps the database the same with cart